src/distortion.js

```diff
@@ -118,38 +125,42 @@ define(function (require) {
     return this.waveShaperNode.oversample;
   }
 
-  /**
-   *  Send output to a p5.sound or web audio object
-   *
-   *  @method connect
-   *  @param  {Object} unit
-   */
-  p5.Distortion.prototype.connect = function(unit) {
-    var u = unit || p5.soundOut.input;
-    this.output.connect(u);
-  };
-
-  /**
-   *  Disconnect all output.
-   *
-   *  @method disconnect
-   */
-  p5.Distortion.prototype.disconnect = function() {
-    this.output.disconnect();
-  };
+  // /**
+  //  *  Send output to a p5.sound or web audio object
+  //  *
+  //  *  @method connect
+  //  *  @param  {Object} unit
+  //  */
+  // p5.Distortion.prototype.connect = function(unit) {
```

Comment:
hmm, we should probably keep documentation like this within each class/module, even though the methods are covered in p5.Effect module. These documentation blocks determine what shows up on https://p5js.org/reference/#/p5.Distortion methods section.

Another way to handle it might be to create documentation for p5.Effect, and then each class that inherits from p5.Effect includes a link in the documentation saying "inherits methods from p5.Effect".